test/system/comments_test.rb

```diff
@@ -3,47 +3,74 @@
 require "application_system_test_case"
 
 class CommentsTest < ApplicationSystemTestCase
-  setup do
-    @comment = comments(:one)
+  include Warden::Test::Helpers
+
+  def setup
```

Comment:
```suggestion
  setup do
```

他にもあるか、提出物の中をすべて見直してくださいー

Reply:
ありがとうございます！

誤って全部逆に修正していたので、修正し直しました！

Reply:
これらの違いはこんな感じです。

[minitestでdef setupとsetup doの違い \- komagataのブログ](http://docs.komagata.org/5203)